.github/workflows/create-tink-keyset-test.yml

```diff
@@ -0,0 +1,32 @@
+name: Create Tink Keyset Test
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Should this also run on pull requests?

Reply:
whoops, changed to only run on `pull_request`